src/Universe.h

```diff
@@ -83,7 +83,7 @@ class Universe {
 
 public:
 
```

Comment:
I believe you also need to set the default const int id for `Lattice` to -1. Otherwise, all lattices where the user doesn't specific the id will have an id of `0`.